backend/common/src/main/java/ai/verta/modeldb/common/connections/UAC.java

```diff
@@ -73,18 +73,22 @@ private UAC(UAC other) {
   }
 
   public UAC withServiceAccount() {
+    return this.withServiceAccount(this.serviceUserEmail, this.serviceUserDevKey);
+  }
+
+  public UAC withServiceAccount(String serviceUserEmail, String serviceUserDevKey) {
     UAC c = new UAC(this);
```

Comment:
why would you name this `c`

Reply:
Probably the type was named something else before.